src/NeuralNetDiffEq.jl

```diff
@@ -58,10 +58,10 @@ function Base.show(io::IO, A::KolmogorovPDEProblem)
   show(io , A.sigma)
 end
 
-struct GeneranNNPDEProblem{PF,BF,S,D,P} <:DiffEqBase.DEProblem
+struct GeneranNNPDEProblem{PF,BF,SP,D,P} <:DiffEqBase.DEProblem
   pde_func::PF
   bound_funcs::BF
-  space ::S
+  space ::SP
```

Comment:
that's not what I meant, I meant there's a spelling error in all of these.

Reply:
oh yeah, thanks